index.html

```diff
@@ -31,7 +31,7 @@ <h1 class="primary-text">
                     <p>
                         I’m a software developer! I can help you build a product ,
```

Comment:
https://github.com/nunegabriel/personal-portfolio/blob/5f39f5b40e68e8f7c5c555e7870a05b5f16616c4/index.html#L72-L74
https://github.com/nunegabriel/personal-portfolio/blob/5f39f5b40e68e8f7c5c555e7870a05b5f16616c4/index.html#L108-L110
https://github.com/nunegabriel/personal-portfolio/blob/5f39f5b40e68e8f7c5c555e7870a05b5f16616c4/index.html#L145-L147


- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻